Automattic/jetpack
# Creates the MC Stats package

This PR creates the MC Stats package. It copies the functionality present in the Jetpack class to its own class, to be used in further PRs.

It adds a class with methods to bump stats to MC Stats.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
p9dueE-1tz-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* This PR just creates the package and don't use it, so nothing should change
* run unit tests by going to the package folder and `composer run phpunit`
* check the code
